lib/tweetsclient/postgres_track.py

```diff
@@ -0,0 +1,56 @@
+import psycopg2
```

Comment:
Where is this being used?

Reply:
It is used [here](https://github.com/CodeForAfrica/politwoops-tweet-collector/blob/dev/conf/tweets-client.ini.example#L13) you specify which module you want to be used for tracking